server/src/config/config.js

```diff
@@ -0,0 +1,12 @@
+/* eslint import/no-dynamic-require: 0 */
+import _ from 'lodash';
```

Comment:
you don't need to import the entire lodash library, they have modularized most of their functions : https://www.npmjs.com/package/lodash.defaultsdeep ;)